website/content/en/preview/upgrade-guide/_index.md

```diff
@@ -36,38 +38,39 @@ When there is a breaking change we will:
 * Increment the minor version when in major version 0
 * Add a permanent separate file named `migrating_to_vx.y.z.md` to our website (linked at the bottom of this page)
   clearly explaining the breaking change and what needs to be done on the customer side to ensure a safe upgrade
-* Add the sentence “This is a breaking change, please refer to `migrating_to_x.y.z.md` for upgrade instructions” to the
-  top of the release notes and in all our announcements
+* Add the sentence “This is a breaking change, please refer to `migrating_to_x.y.z.md` for upgrade instructions” to the top of the release notes and in all our announcements
 
 ## How Do We Find Incompatibilities
 
 Besides the peer review process for all changes to the code base we also do the followings in order to find
 incompatibilities:
-* (To be implemented) Automated tests to see if installing, uninstalling, upgrading from an older version and downgrading
-to an older version to check the compatibility of the application
-* (To be implemented) Turn the commands in our
-documentation into scripts that we can automatically run to find incompatibilities between the application and
-documentation
+* (To be implemented) Automated tests for installing, uninstalling, upgrading from an older version, and downgrading to an older version to check the compatibility of the application
+* (To be implemented) Turn the commands in our documentation into scripts that we can automatically run to find incompatibilities between the application and documentation
```

Comment:
May be this should be "to check the compatibility of the documentation with the application" to be consistent with the bullet point